corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
+        simple_transformations = {
+            EditActionType.REPLACE: lambda x: self.replace_string,
```

Comment:
@biyeun The [spec](https://docs.google.com/document/d/1mboGuJDQmC3zREoGtWmUHRrZyDX7S1JlEZ7HSuLowfc/edit?tab=t.0) says `replace_all_string` will be used with the `REPLACE` action, but it doesn't describe `replace_string` at all. What's the intended difference between `replace_string` and `replace_all_string`?

Reply:
`self.replace_all_string` is what should be used here

Reply:
What's the purpose of `repalce_string`?

Reply:
`self.replace_string` is used in find and replace. I can combine the two if you feel bothered by it, but the two will be separate fields in the form

Reply:
Yes, I think they should be combined.